src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/Controllers/TeamPlanController/ErrorBanner/ErrorBanner.tsx

```diff
@@ -37,7 +34,8 @@ export default function ErrorBanner({
   const { data: accountDetails } = useAccountDetails({ provider, owner })
   const { proPlanYear } = findProPlans({ plans })
   const { sentryPlanYear } = findSentryPlans({ plans })
-  const plan = accountDetails?.rootOrganization?.plan ?? accountDetails?.plan
+  const plan =
```

Comment:
this was a bug :)

Reply:
Nice catch!

Reply:
Was TS helpful here in pointing this out, or was it just something you noticed?

Reply:
A little bit of both tbh, it wasn't until I had full types for everything and unwound what this was actually trying to do that I was able to figure out what the right param was supposed to be